web/packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -194,6 +190,14 @@ export function SelectResource({ onSelect }: SelectResourceProps) {
                       onSelect(r);
                     }
                   },
+                  onKeyUp: (e: KeyboardEvent) => {
+                    if (e.key === 'Enter' && r.hasAccess) {
+                      setShowApp(true);
+                      onSelect(r);
+                    }
+                  },
+                  tabIndex: 0,
+                  role: 'button',
```

Comment:
That's a good direction, but I'd go even further and change the type of `ResourceCard` to `button` (it will require some style adjustments).